### ISSUE TITLE
Overwrite vs Add Calendar Itesm

### ISSUE THREAD

**Issue by srgeek78** [David-Barrett-MS/PowerShell-EWS-Scripts#16]
Can you enhance your powershelgl script to allow for overwriting based on Subject and Start Date being the same as what that person has in their calendar already? We have a scenario where we are pushing the company holidays to everyone's calendars. Your script works awesome to bring it in the first time but I don't want to rerun the script and get duplicates. I would rather it ignore it if it already exists. It could simply be a flag that doesn't allow it to place duplicates on the calendar if it already exists.

Thanks,
Jason

**Comment by David-Barrett-MS** [comment]
This is for the Import-CalendarCSV.ps1 script?

**Comment by srgeek78** [comment]
That is correct. JasonOn Nov 30, 2022, at 10:20, David Barrett ***@***.***> wrote:﻿
This is for the Import-CalendarCSV.ps1 script?
